packages/circularprogress/src/react/__specs__/index.spec.tsx

```diff
@@ -11,7 +11,10 @@ describe('CircularProgress', () => {
 
   it('renders', () => {
     const { getByTestId } = render(
-      <CircularProgress data-testid="mock-component" />
+      <CircularProgress
+        data-testid="mock-component"
+        aria-label="ciruclar progress"
```

Comment:
```suggestion
        aria-label="circular progress"
```

Reply:
No clue honestly I used what is available to me to use https://chrome.google.com/webstore/detail/screen-reader/kgejglhpjiefppelpmljglcjbhoiplfn. I also followed the Deque pattern. Once I got it reading what I wanted I stopped.